docs/source/python/extending_types.rst

```diff
@@ -290,21 +290,25 @@ Custom scalar conversion
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you want scalars of your custom extension type to convert to a custom type when
-:meth:`ExtensionScalar.as_py()` is called, you can override the :meth:`ExtensionType.scalar_as_py()`
+:meth:`ExtensionScalar.as_py()` is called, you can override the :meth:`ExtensionScalar.as_py()`
```

Comment:
```suggestion
:meth:`ExtensionScalar.as_py()` is called, you can override the :meth:`ExtensionScalar.as_py()` by subclassing :class:`ExtensionScalar`.
```

Reply:
Whoops, my last inline comment was not fully resulting in a correct word order -> https://github.com/apache/arrow/pull/13649